index.html

```diff
@@ -3,10 +3,28 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="estilos.css">
+    <link href="https://fonts.googleapis.com/css2?family=Pangolin&family=Yanone+Kaffeesatz:wght@500;700&display=swap" rel="stylesheet">
     <title>Document</title>
 </head>
+
 <body>
-    <h1>Hola Dani, que comience el juego...</h1>
+    <!-- Voy con mi vedette la hora : ) -->
+    <div class="contenedor_fecha">
+        
+        Hoy es un gran día,&nbsp;<div id="diaSemana"></div>&nbsp;<div id="dia"></div>&nbsp;de&nbsp;<div id="mesLetra"></div>
     
+    </div>
+
+    <h1>Hola Dani, que comience el juego...</h1>
+    <h2>Esta es otra rama</h2>
+ <br>
+ <br>
+ <br>
+ <br>
+ <br>
+ <br>
+<!-- <p>Dani, me habia trabado ayer en un error lo llame a la hoja de app.js como un link y me tiraba este error del servidor   Refused to apply style from 'http://127.0.0.1:5500/static/app.js' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.  porque yo habia puesto esto <link rel="stylesheet" href="app.js"> y le saque el rel no me tiraba  ya mas el error, pero no me la habilitaba la pagina entonces si o si tuve que  hacer esto  <script src="app.js"></script>  </p> -->
```

Comment:
Mmm, me parece que el link es solo cuando queres linkear otra pagina, no a un archivo javascript. Para eso siempre lo tenes que cargar con el tag script.
Si quisieras tener un link a una segunda pagina, tendrias que tener otro archivo html y en ese archivo html el script javascript de esa pagina, y ahi si podrias hacer un <link href='segunda_pagina.html'>

Lo que si veo que podes tener buscando un poco, es archivos "preload". Por ejemplo en esta pagina podrias tener app.js y ademas tener otro archivo .js para cargar otras cosas, entonces ademas del script que agregaste, arriba podrias tener:
```
  <link rel="preload" href="style.css" as="style">
  <link rel="preload" href="main.js" as="script">
```

Pero fijate que igual le pone el `as="script"`, que significa que lo va a cargar como si fuera un script, qeu es necesario para js

Reply:
con ese link no me lo permite intente reemplazarlo por el <scricpt> app.js </script>, pero no me funciono. me aparecio este error en el servidor 
           Failed to load resource: the server responded with a status of 404 (Not Found)

Reply:
Era solo un ejemplo eso para que vieras la sintaxis. Vos no tenes en tu proyecto main.js o style.css. Sino poneme exactamente que es lo que agregaste.